dtfs-central-api/api-tests/mocks/gef/gef-applications.js

```diff
@@ -0,0 +1,230 @@
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 9',
+  additionalRefName: 'Team 9',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 10',
+  additionalRefName: 'Team 10',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 11',
+  additionalRefName: 'Team 11',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 12',
+  additionalRefName: 'Team 12',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 13',
+  additionalRefName: 'Team 13',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 14',
+  additionalRefName: 'Team 14',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 15',
+  additionalRefName: 'Team 15',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+},
+{
+  dealType: 'GEF',
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 1',
+  additionalRefName: 'Team 1',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 1,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}];
+
+module.exports = APPLICATION;
```

Comment:
can we use a "base deal" and extend that for each different type of mock deal?

Reply:
done